flexmeasures/data/models/audit_log.py

```diff
@@ -178,3 +185,33 @@ def add_record(
             affected_asset_id=asset.id,
         )
         db.session.add(audit_log)
+
+
+def truncate_event(
+    event: str, attr_key: str, old_value: str, new_value: str, max_length: int = 250
+) -> str:
+    """Truncate the event string and place ellipses in the middle if it exceeds max_length."""
+
+    # Truncate values only if event length exceeds max_length
+    if len(event) > max_length:
+        truncated_new_value = truncate_string(str(new_value), 60)
```

Comment:
Again, 60 is a heuristic - how did you choose it?

Reply:
I accept the general approach. I see how making it a bit more dynamic might be nice, but it's also not the most important feature, so don't spend too much time here.

Reply:
okay, yes i understand.